addons/hr_holidays/migrations/14.0.1.6/pre-migration.py

```diff
@@ -0,0 +1,7 @@
+def migrate(cr, version):
+    cr.execute("""
+        UPDATE hr_leave
+        SET request_unit_custom = FALSE
+        WHERE request_unit_custom = TRUE 
+        AND  holiday_status_id IS NOT NULL OR request_unit_half OR request_unit_hours;
```

Comment:
Cụm sau AND và cho vào trong ngoặc chứ. Không chắc nó có tương đương hay không như cho vào ngoặc đọc code dễ hiểu hơn.

Reply:
@davidtranhp dạ vâng anh, để em sửa lại

Reply:
Sau AND thừa dấu cách trống thì phải @hoangtiendung070797 ?

Reply:
> Sau AND thừa dấu cách trống thì phải @hoangtiendung070797 ?

@leotranviindoo dạ vâng anh, em sửa lại r push lên ạ